zerver/views/digest.py

```diff
@@ -7,14 +7,29 @@
 from django.utils.timezone import now as timezone_now
 
 from zerver.decorator import zulip_login_required
+from zerver.forms import DigestParameterForm
 from zerver.lib.digest import DIGEST_CUTOFF, get_digest_context
 
 
 @zulip_login_required
 def digest_page(request: HttpRequest) -> HttpResponse:
     user_profile = request.user
+
     cutoff = time.mktime((timezone_now() - timedelta(days=DIGEST_CUTOFF)).timetuple())
+    message_weight = 1
+    sender_weight = 1
+
+    if request.method == "POST":
+        form = DigestParameterForm(request.POST)  # nocoverage
+
+        if form.is_valid():  # nocoverage
+            message_weight = form.cleaned_data["message_weight"]  # nocoverage
+            sender_weight = form.cleaned_data["sender_weight"]  # nocoverage
```

Comment:
So, you can do `# nocoverage` on the whole block, and that's much better than adding it on every line.  But this begs the question -- why not just add a test for this?

Reply:
Sure, I would add a test.